Doc/library/pathlib.rst

```diff
@@ -490,7 +490,7 @@ Pure paths provide the following methods and properties:
       True
 
 
-.. method:: PurePath.is_relative_to(*other)
+.. method:: PurePath.is_relative_to(other)
```

Comment:
Need to add an appropriate deprecation warning about the change in call signature so folks coming from older code don't wonder about the change.